.github/workflows/manual-review.yml

```
@@ -1,6 +1,6 @@
 name: Manual Review
 
-on: [pull_request]
+on: [push]
```

Since there's no review on `master`:

```suggestion
on:
  push:
    branches-ignore:
      - master
```